www/layouts/newsletter/section.html

```diff
@@ -0,0 +1,465 @@
+            <option value="Tunisia">Tunisia</option>
+            <option value="Turkey">Turkey</option>
+            <option value="Turkmenistan">Turkmenistan</option>
+            <option value="Turks &amp; Caicos Islands"
+              >Turks &amp; Caicos Islands</option
+            >
+            <option value="Turks and Caicos Islands"
+              >Turks and Caicos Islands</option
+            >
+            <option value="Tuvalu">Tuvalu</option>
+            <option value="Uganda">Uganda</option>
+            <option value="Ukraine">Ukraine</option>
+            <option value="United Arab Emirates">United Arab Emirates</option>
+            <option value="United Kingdom">United Kingdom</option>
+            <option value="Uruguay">Uruguay</option>
+            <option value="USA Minor Outlying Islands"
+              >USA Minor Outlying Islands</option
+            >
+            <option value="Uzbekistan">Uzbekistan</option>
+            <option value="Vanuatu">Vanuatu</option>
+            <option value="Vatican City State (Holy See)"
+              >Vatican City State (Holy See)</option
+            >
+            <option value="Venezuela">Venezuela</option>
+            <option value="Vietnam">Vietnam</option>
+            <option value="Virgin Islands (British)"
+              >Virgin Islands (British)</option
+            >
+            <option value="Virgin Islands (U.S.)">Virgin Islands (U.S.)</option>
+            <option value="Wallis and Futuna Islands"
+              >Wallis and Futuna Islands</option
+            >
+            <option value="Western Sahara">Western Sahara</option>
+            <option value="Yemen">Yemen</option>
+            <option value="Zambia">Zambia</option>
+            <option value="Zimbabwe">Zimbabwe</option>
+          </select>
+        </div>
+        <div id="mce-responses" class="clear">
+          <div
+            class="response"
+            id="mce-error-response"
+            style="display:none"
+          ></div>
+          <div
+            class="response"
+            id="mce-success-response"
+            style="display:none"
+          ></div>
+        </div>
+        <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+        <div style="position: absolute; left: -5000px;" aria-hidden="true">
+          <input
+            type="text"
+            name="b_ad81d725159c1f322a0c54837_4c04dfddc5"
+            tabindex="-1"
+            value=""
+          />
+        </div>
+        <div class="clear pt-3">
+          <input
+            type="submit"
+            value="Submit"
+            name="subscribe"
+            id="mc-embedded-subscribe"
+            class="button"
+          />
+        </div>
+      </div>
+    </form>
+  </div>
+  <script
+    type="text/javascript"
+    src="//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js"
+  ></script>
+  <script type="text/javascript">
+    ;(function($) {
```

Comment:
Do we need to move this js in your `mailchimp.js` file? So this section.html file has only html code

Reply:
mailchimp.js is actually loaded on every page so I think its better to keep all the mail chimp validation in the same file to not cause any scoping issues

Reply:
I think ideally specific js files should be loaded in their respective files only (where they are required) but since this approach is not followed for other js files too so we can keep it like this for now. Maybe we improve this later on.